### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by yuvenalmash**
## Changes summary
- Style the application to match the given sample
- Change css to scss

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -19,23 +19,29 @@ const Book = (props) => {
...
         </div>
       </section>
+      <hr />
       <section className={styles.col2}>
         <p>CURRENT CHAPTER</p>
-        <p>{currentChapter}</p>
+        <h3>{currentChapter}</h3>
         <button type="button">UPDATE PROGRESS</button>
       </section>
     </div>
```
Comment (TadesseDev) [inline src/components/Book.js:47]: - [ ] Dear @yuvenalmash, Your design is already impressive, but there are some minor improvements to be added to make your design even better. One is that your buttons need some padding from the right and left, also the font color better be a bit darker to look exactly like the Zeplin design. 
### Zeplin Design
Screenshot 2023-01-28 173758

### Your design 
Screenshot 2023-01-28 173812

File: src/components/AddBook.js [hunk 2]
```diff
@@ -30,26 +37,35 @@ const AddBook = () => {
...
+          ref={(node) => {
+            category = node;
+          }}
+        >
+          <option value="action">Action</option>
+          <option value="scifi">Science Fiction</option>
+          <option value="fantasy">Fantasy</option>
+        </select>
+        <button type="submit">ADD BOOK</button>
+      </form>
```
Comment (TadesseDev) [inline src/components/AddBook.js:67]: - [ ] Dear @yuvenalmash, For your form I suggest you make it stretch to the whole page as shown in the Zeplin design, also the border should be a bit lighter.

### Zeplin Design
Screenshot 2023-01-28 173610
### Your design
Screenshot 2023-01-28 173627

File: src/components/BooksContainer.js [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
 import AddBook from './AddBook';
+import styles from './css/BooksContainer.module.scss';
 
 const BooksContainer = () => (
   <div>
     <BooksList />
+    <hr className={styles.separator} />
     <AddBook />
   </div>
 );
```
Comment (TadesseDev) [inline src/components/BooksContainer.js]: - [ ] Dear @yuvenalmash, I'm impressed with your initial design, But I believe there are possible improvements we can make. as you can see in the Zeplin design the body color is a bit darker and the books/cards are lighter. I see you use the white color for the body and card and apply a border for the cards, this is not bad but to make your design even better I suggest you do differentiate the color between the body background and the background of the cards.

- [ ] Also, consider making the active navigation item a bit bolder and the inactive one a bit lighter. 
### Zeplin Design 
Screenshot 2023-01-28 174822

### Your design
Screenshot 2023-01-28 174900

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @yuvenalmash 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7

## Highlights
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitHub Flow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅

## Hi @yuvenalmash 👋

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**



## Highlights:
* Well-written PR description ✔️ 
* All tests passed ✔️ 
* The app looks nice 💯 

## Optional suggestions:

- [ ] Please consider using best practices when you write commit messages, they should be capitalized sentences 👍 


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**